app/assets/v2/js/shared.js

```diff
@@ -825,7 +825,11 @@ var promptForAuth = function(event) {
 
     token_contract.allowance.call(from, to, function(error, result) {
       if (error || result.toNumber() == 0) {
-        _alert("You have not yet enabled this token.  To enable this token, go to the <a style='padding-left:5px;' href='/settings/tokens'> Token Settings page and enable it</a>. (this is only needed one time per token)");
+        if (!document.alert_enable_token_shown) {
+          _alert("You have not yet enabled this token.  To enable this token, go to the <a style='padding-left:5px;' href='/settings/tokens'> Token Settings page and enable it</a>. (this is only needed one time per token)");
+        }
+        document.alert_enable_token_shown = true;
```

Comment:
I think it should be a warning alert like [here](https://github.com/gitcoinco/web/blob/639b0de163aa2a7d251efe6d605492264e9c9b6f/app/assets/onepager/js/send.js#L115)